src/Inkluzitron/Modules/BdsmTestOrg/OpenSansFontManager.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Linq;
+using System.Drawing.Text;
+using System.Drawing;
+using System.Runtime.InteropServices;
+
+namespace Inkluzitron.Resources.Fonts.OpenSans
+{
+    public class OpenSansFontManager : IDisposable
```

Comment:
Myšlenka: Udělat z toho Dictionary like font manager 🤷 , případně service, jak se to řešilo na DC.